app/code/core/Mage/Adminhtml/controllers/TaxController.php

```diff
@@ -28,13 +28,7 @@ public function ignoreTaxNotificationAction()
     {
         $section = $this->getRequest()->getParam('section');
         if ($section) {
-            try {
-                $path = 'tax/ignore_notification/' . $section;
-                Mage::getModel('core/config')->saveConfig($path, 1);
-                Mage::getConfig()->reinit();
-            } catch (Exception $e) {
-                Mage::getSingleton('adminhtml/session')->addError($e->getMessage());
-            }
+            Mage::helper('tax')->setIsIgnored('tax/ignore_notification/' . $section, 1);
```

Comment:
`declare(strict_types=1);`
Fatal error: Uncaught TypeError: test(): Argument #1 ($value) must be of type bool, int given
To prevent the above:
```suggestion
            Mage::helper('tax')->setIsIgnored('tax/ignore_notification/' . $section, true);
```